Phase05/Phase05-Console/Phase05-Console/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using InvertedIndex;
+
+namespace Phase05_Console;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        var path = @"C:\Users\Arya\OneDrive\Desktop\are\Summer1401-SE-Team08\Phase05-Console\EnglishData";
+        InvertedIndex.InvertedIndex index = new InvertedIndex.InvertedIndex(path);
+        var result = index.SearchDocsForQuery("God");
```

Comment:
Why is your app like this?
The user is supposed to be able to make a query, not just see the result of a query you have made for them :)